src/components/Map.js

```diff
@@ -10,17 +14,27 @@ function Map(){
         zoom: 11
 
     };
+
+    useEffect(()=> {
+        fetchNearbyPlaces(defaultLocation.center.lat, defaultLocation.center.lng).then(res => setNearbyPlaces(res));
+    }, []);
+    
     return (
         <div style={{ height: 'calc(66.67vh - 1.25rem)', width: '100%' }}>
           <GoogleMapReact
             bootstrapURLKeys={{ key:'AIzaSyA_jF-TPUl8qTMZ3BKFTrFOolH9wR7NOz4'}}
             defaultCenter={defaultLocation.center}
             defaultZoom={defaultLocation.zoom}
             >
-            
+            {nearbyPlaces?
+                (nearbyPlaces.map(
+                    ({latitude,longitude,title,description,image}, index) => <Pin key={index} lat={latitude} lng={longitude} title={title} description={description} image={image} />))
+                :
+                null
+            }
```

Comment:
A great way to help make this even more concise that I THINK should still work would be using the "logical && operator", which would essentially replace the `?` with `&&` and allow us to eliminate lines 32 and 33. Total preference thing though.